dark/reads.py

```diff
@@ -711,7 +768,7 @@ def maximumORFLength(self):
         read. The ORF may originate or terminate outside the sequence, which is
         why the length is just a lower bound.
         """
-        return max(len(orf) for orf in self.ORFs())
+        return max(len(orf) for orf in self.ORFs(True))
```

Comment:
Passing `True` like this is not a good idea if that's already the default value of that argument (I think the arg is to allow open ORFs). The default might change in the future (e.g., someone might change the variable name to `disallowOpenORFs` and change the default to `True`, in which case this code will break and/or need updating). There's already a sensible default on that function, so either allow someone to pass *you* a value which you then pass on, or else step out of the way and don't silently impose your inflexible imperialistic worldview on two innocent pieces of code that are simply trying to get along.

Reply:
Changed so that `openORFs` can be specified when calling the function. Updated the tests for `maximumORFLength` and added another Test where `False` is given.